profiles/tinoschroeter.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://raw.githubusercontent.com/tinoschroeter/tinoschroeter/master/tinoschroeterGitHub.png",
```

Comment:
I suggest a image like this:

```suggestion
  "image": "https://user-images.githubusercontent.com/91655303/150111733-c805d10b-b465-4d55-8389-446da75b1e61.png",
```


![image](https://user-images.githubusercontent.com/91655303/150111774-47566387-196d-42d3-b9ae-4bc19f8254d5.png)